wp-api.js

```diff
@@ -1,24 +1,19 @@
-(function( window, undefined ) {
+'use strict';
```

Comment:
This ought to go inside the IIFE—this will impact other included scripts if somebody uses a plugin to concatenate this with any of their other JS dependencies.